eas.json

```diff
@@ -47,11 +47,13 @@
     "developmentBuild": {
       "distribution": "internal",
```

Comment:
```suggestion
    "developmentBuild": {
      "extends": "base",
      "distribution": "internal",
```

Reply:
And add a "base" profile on top, with `image: latest` for iOS and Android

Reply:
Maestro doesn't work on all images, at least on iOS. I prefer to keep specific images versions so tag upgrades (changing `default` or `latest`) don't automatically, without us knowing, break the workflow. We should test the workflow on new images, hoping upcoming Xcode versions unbreak Simulator.